source/core/slang-shared-library.cpp

```diff
@@ -87,4 +99,50 @@ void* DefaultSharedLibrary::findSymbolAddressByName(char const* name)
     return SharedLibrary::findSymbolAddressByName(m_sharedLibraryHandle, name);
 }
 
+
+String SharedLibraryUtils::getSharedLibraryFileName(void* symbolInLib)
```

Comment:
Having this return a SlangResult and take a StringBuffer for output, does have the benefit of the function being to communicate more about it's failure. Admittedly it might be a bit clumsier to use.